README.md

```diff
@@ -1,191 +1,58 @@
-}
-
-// Attach the chat room - this means we will begin to receive messages from the server
-try await room.attach()
-
-// Add a listener for new messages in the chat room
-let messagesSubscription = try await room.messages.subscribe()
-Task {
-    // Subscribe to messages
-    for await message in messagesSubscription {
-        print("Message received: \(message.text)")
-    }
-}
+## Supported Platforms
 
-// Send a message
-_ = try await room.messages.send(
-    params: .init(text: "Hello, World! This is my first message with Ably Chat!"))
+Ably aims to support a wide range of platforms. If you experience any compatibility issues, open an issue in the repository or contact [Ably support](https://ably.com/support).
 
-// Wait 5 seconds before closing the connection so we have plenty of time to receive the message we just sent
-// This disconnects the client from Ably servers
-try await Task.sleep(nanoseconds: 5 * NSEC_PER_SEC)
-await chatClient.rooms.release(roomID: "readme-getting-started")
-realtime.close()
-print("Connection closed")
-```
+This SDK supports the following platforms:
 
-Now run your script:
+| Platform | Support      |
+|----------|--------------|
+| iOS      | >= 14.0      |
+| macOS    | >= 11.0      |
+| tvOS     | >= 14.0      |
 
-```shell
-swift run
-```
+> [!NOTE]
+> Xcode v16.1 or later is required.
 
-All being well, you should now see the following in your terminal:
-
-```
-Room status changed: attaching(error: nil)
-Connection status changed: connected
-Room status changed: attached
-Message received: Hello, World! This is my first message with Ably Chat!
-Room status changed: releasing
-Room status changed: released
-Connection closed
-```
+> [!IMPORTANT]
+> SDK versions <  1.2.24 will be [deprecated](https://ably.com/docs/platform/deprecate/protocol-v1) from November 1, 2025.
 
-Congratulations! You have sent your first message using the Ably Chat SDK!
-
-## Example app
+---
 
-This repository contains an example app, written using SwiftUI, which demonstrates how to use the SDK. The code for this app is in the [`Example`](Example) directory.
-
-In order to allow the app to use modern SwiftUI features, it supports the following OS versions:
+## Releases
 
-- macOS 14 and above
-- iOS 17 and above
-- tvOS 17 and above
+The [CHANGELOG.md](/ably/ably-chat-swift/blob/main/CHANGELOG.md) contains details of the latest releases for this SDK. You can also view all Ably releases on [changelog.ably.com](https://changelog.ably.com).
 
-To run the app, open the `AblyChat.xcworkspace` workspace in Xcode and run the `AblyChatExample` target. If you wish to run it on an iOS or tvOS device, you’ll need to set up code signing.
+---
 
-## Contributing
+## Contribute
 
-For guidance on how to contribute to this project, see the [contributing guidelines](CONTRIBUTING.md).
+Read the [CONTRIBUTING.md](./CONTRIBUTING.md) guidelines to contribute to Ably or [Share feedback or request](https://forms.gle/mBw9M53NYuCBLFpMA) a new feature.
```

Comment:
```suggestion
Read the [CONTRIBUTING.md](./CONTRIBUTING.md) guidelines to contribute to Ably or [share feedback or request a new feature](https://forms.gle/mBw9M53NYuCBLFpMA).
```

Reply:
I think that the link looks a bit odd as-is:

- why is "Share"  capitalised?
- the link ends abruptly in the middle of a sentence; if "share feedback" is part of the link, why wouldn't "request a new feature" be?